setup.sh

```diff
@@ -4,21 +4,27 @@ jarvispath=$PWD
 touch jarvis
 chmod +rwx jarvis
 exec 3<> jarvis
-
     echo "#!/bin/bash" >&3
     echo "source $jarvispath/env/bin/activate" >&3
     echo "python $jarvispath/jarviscli/" >&3
```

Comment:
For python3 we should run jarviscli with python3 command

Reply:
Since we are creating a virtual environment and once we activate this environment python runs in python version of the virtual environment created.
Thus I have added a command python --version to verify that it is indeed the case